article.Rmd

```diff
@@ -194,7 +196,7 @@ m1
 
 <!-- The region of analysis may seem like a basic consideration: most cities have well-defined administrative zones. -->
 <!-- In Leeds and many other cities, it makes sense to focus on the region directly surrounding the city centre, in a kind of 'geographical triage' to omit from the analysis pop-up options in the outskirts and focus valuable attention on the routes that are most likely to serve the highest number of people. -->
-We set a modifiable parameter `city_centre_buffer_radius` with an initial value of 8 km (5 miles) to geographically subset potential routes.
+We set a modifiable parameter `city_centre_buffer_radius` with an initial value of 8 km (5 miles) to geographically subset potential routes. 5 miles represents a distance that most people have the physical ability to cycle.
```

Comment:
OK but needs a reference. Do you have one @mvl22 ?

Reply:
Yes, I realised subsequently that it needs a reference, even though it's probably common sense to most people. I don't know of a suitable reference to hand. Is there something in the PCT dataset that might indicate this?